lib/open_invoice/client.rb

```diff
@@ -31,9 +31,5 @@ def supplier
     def buyer
       @buyer ||= ::OpenInvoice::Entities::Buyer.new(supplier_uuid)
     end
-
```

Comment:
I've seen payment API used by Quickbooks integration. Is it possible that Braavos calls `payment` for all the integrations and might error out if this is removed? (just wondering if this should be a no-op rather than removed)

Reply:
hm it is possible @dbaron-ft. We have no payments in our braavos db where the source is oi, but looking at the data import job in braavos, there is a Generics::Importers::Base class that is used to imort payments.. which makes me think that maybe it tries to call this endpoint, even for OI?

I can leave it in & add a comment if that makes more sense?

Reply:
Some more context, I met with the OI API team last week and asked them about this endpoint. The investigated and confirmed that they have nothing on their end related to a payment endpoint currently in production or being developed